src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.constants.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
```

Comment:
You need Color for your figures. Why do you use `name()` here to return String and then use `Color.valueOf` when create figures?
Just return Color here.

Reply:
ColorSupplier with public String getRandomColor()
It is a requirement from README

Reply:
Just make color field in Figure of type String))